### PR TITLE
Add py2700 Recipe

### PR DESCRIPTION
**Pull request by ngpaladi**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ngpaladi** [comment]
@conda-forge/help-python

**Comment by ngpaladi** [comment]
I'm confused as to why this is failing - it seems to build fine for me on my machine and I can install from PyPI without issues

**Comment by synapticarbors** [comment]
If you look at the logs (https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=215224&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=705) you can see that conda-forge doesn't have a new enough version of pyvisa to satisfy `pip`. It appears that work is underway to fix that: https://github.com/conda-forge/pyvisa-feedstock/pull/22

File: recipes/py2700/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ngpaladi/py2700
+  summary: 'A Python package to interface with a Keithley 2700 multimeter'
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/py2700/meta.yaml:40]: I've opened a PR upstream to address manually packaging the license file:
https://github.com/ngpaladi/py2700/pull/1

Once it is merged or if some other solution is used and a new release is spun out, then the manually packaged license file should be removed from the feedstock.

Comment (ngpaladi) [reply recipes/py2700/meta.yaml:40]: Thank you, I will merge that and create a new release promptly. I can see if an older version of pyvisa works as well, or just wait for the update to pyvisa.

**Comment by ngpaladi** [comment]
The license is now bundled with the source and I have a released a new version accordingly.

**Comment by ngpaladi** [comment]
@conda-forge-admin please rerender

**Comment by dopplershift** [comment]
Looks like this recipe (or directory) was originally added in #12754, so you'll need to take care of the conflict here.

**Comment by ngpaladi** [comment]
Thank you, will do. It seems the two got mixed up, and the other went through fine. I'll re-fork and try again.

**Pull request closed without merging**